src/foobar/models.py

```diff
@@ -92,7 +92,7 @@ class WalletLogEntry(UUIDModel, TimeStampedModel):
     user = models.ForeignKey(User, null=True, blank=True)
     trx_type = EnumIntegerField(enums.TrxType,
                                 default=enums.TrxType.CORRECTION)
-    wallet = models.ForeignKey('wallet.Wallet', related_name='log_entries')
+    wallet = models.ForeignKey('wallet.Wallet', related_name='log_entries', editable=False)
```

Comment:
Just a heads up, because settings `editable=False` should trigger a migration. In case it does, we are missing a migration file, test running `python manage.py makemigrations` to see if it generates a new migration file.

For the admin panel, if you haven't figured it out already, the same effect as setting `editable=False` here would be applied by including the field in the ModelAdmins `readonly_fields`.